src/mainboard/amd/romecrb/src/main.rs

```diff
@@ -415,15 +409,12 @@ fn start_bootstrap_core(fdt_address: usize) -> ! {
         msrs(w);
     }
     c00(w);
-    write!(w, "LDN is {:x}\r\n", peek32(0xfee000d0)).unwrap();
-    poke32(0xfee000d0, 0x1000000);
-    write!(w, "LDN is {:x}\r\n", peek32(0xfee000d0)).unwrap();
-    write!(w, "loading payload with fdt_address {}\r\n", fdt_address).unwrap();
 
     boot(w, fdt_address);
 
-    write!(w, "Unexpected return from payload\r\n").unwrap();
-    arch::halt()
+    loop {
```

Comment:
Could you elaborate a bit more on the `loop`? I'm not sure I understand why that would be necessary. Doesn't `halt()` guarantee that the machine stops?

Reply:
Oops, you are right, that‘s oreboot‘s halt fn which already uses the HLT instruction in an endless loop. I‘ll remove this chunk here on monday.